packages/jokul/src/components/button/styles/button.scss

```diff
@@ -125,6 +125,10 @@
 
     &:hover {
         background-color: color-mix(in srgb, var(--background-color), var(--text-color) 15%);
+
+        @media (forced-colors: active) {
```

Comment:
Burde disse også gjelde for `prefers-contrast: more` og `prefers-contrast: custom`? Ser ut som [sistnevnte](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-contrast#custom) overlapper med `forced-colors` i hvert fall.

Reply:
Jeg tror ikke OSet "griper inn" og tuller med systemfarger når man bruker `prefers-contrast`, det hadde i hvert fall ingen slik effekt på min maskin.

Sånn sett burde det kanskje sees på mer generelt dersom Jøkul eventuelt skal støtte det? Kanskje man kunne gjort noe artig med det nye fargesystemet dere holder på med? Det høres jo ut som det burde kunne brukes til noe sånt?